src/lib.rs

```diff
@@ -79,6 +79,7 @@ pub fn run(cmd: Command) -> Result<(), Error> {
         }
         SubCommand::ShowDeps {} => venv_manager.show_deps(),
         SubCommand::ShowVenvPath {} => venv_manager.show_venv_path(),
+        SubCommand::ShowVenvBin {} => venv_manager.show_venv_bin(),
```

Comment:
Maybe `ShowVenvBinPath` and `show_venv_bin_path` ?

That'd be consistent with the `get_venv_bin_path(&self)` you have below